pkgs/servers/hydron/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, ffmpeg-full, graphicsmagick
+, quicktemplate, go-bindata, easyjson }:
+
+buildGoPackage rec {
+  name = "hydron-unstable-${version}";
+  version = "2018-07-15";
+  rev = "3906ace0b4cf48ba9acccf372377c7feb0665be4";
+  goPackagePath = "github.com/bakape/hydron";
+  goDeps = ./deps.nix;
+
+  src = fetchFromGitHub {
+    inherit rev;
```

Comment:
Nit: Place the `rev` definition here, because it's not needed as a drv attribute.